### ISSUE TITLE
Info about hyperparameters

### ISSUE THREAD

**Issue by harshadeepg** [thomasfermi/Dynamic-Coattention-Network-for-SQuAD#1]
Hey! Could you share the hyper parameters you were using to achieve the mentioned EM and F1

**Comment by thomasfermi** [comment]
Hi! As I stated in the README, I achieved the mentioned EM and F1 via 

python code/train.py --rnn_state_size=150

This means that the state size of the RNN was 150, and all other hyperparameters had its "default value". If you check code/train.py you will see lines like

`tf.app.flags.DEFINE_string("model", "DCN", "Choose which model to use baseline/DCN")
tf.app.flags.DEFINE_float("learning_rate", 0.001, "Learning rate.")
tf.app.flags.DEFINE_integer("batch_size", 64, "Batch size to use during training.")
tf.app.flags.DEFINE_integer("epochs", 20, "Number of epochs to train.")`

which tell you what these default values are.
